erc7824-docs/docs/guides/migration-guide.md

```diff
@@ -0,0 +1,708 @@
+---
+sidebar_position: 2
+title: Migration Guide
+description: Guide to migrate to newer versions of Nitrolite
+keywords: [migration, upgrade, breaking changes, nitrolite, erc7824]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## 0.3.0 Changelog
```

Comment:
This is going to be inside release, right?

Just don't forget to remove it before merging

Reply:
317916d2e57ad6cac81303b6526873256e736392